### PR TITLE
WIE: Athena images added

### LINKED ISSUE #180: Add Information about Women's Hackathon on WiE Page
- [x] Add Poster for the Hackathon
- [x] Small Description about the event
- [x] Images related to the event from the Google Photos Link

### PR DESCRIPTION
**Pull request by SaurabhAgarwala**
Few images of the event have been added. This closes #180.

### CODE REVIEW

**Comment by salman-bhai** [comment]
@SaurabhAgarwala is this still relevant? If yes kindly ask Webmaster to Merge!

File: wie.html [hunk 1]
```diff
@@ -128,6 +128,14 @@ <h4>Athena - Women's Hackathon
 					<p class="sans">
 							A 24 hour hackathon was organized for the girl students of NITK under IEEE WiE Affinity Group NITK in association with Web Enthusiasts Club, NITK. The applications developed were all based on a given topic and they were judged by invited professors on the following day and winners were rewarded.
 					</p>
+					<p>
+						<br>
+						<img style="width: 45%; margin: 0% 1%;" src="res/img/athena1.jpg" alt="athena1">
+						<img style="width: 45%; margin: 0% 1%;" src="res/img/athena3.jpg" alt="athena3">
+						<br>
+						<img style="width: 45%; margin: 0% 1%;" src="res/img/athena2.jpg" alt="athena2">
+						<img style="width: 45%; margin: 0% 1%;" src="res/img/athena4.jpg" alt="athena4">
```
Comment (salman-bhai) [inline wie.html]: Preferably define a CSS Class rather than putting css inline.

**Comment by SaurabhAgarwala** [comment]
@salman-bhai Requested changes has been carried out.
@nimbus98 Please review and merge if it looks fine to you.
This closes #180

**Comment by salman-bhai** [comment]
@nimbus98 approved form my end

**Comment by nimbus98** [comment]
Approved from my end as well, but I do not have merging privileges :p 
@anumehaagrawal  Please merge :p

**Pull request merged**